<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeHistoryDialog</class>
 <widget class="QDialog" name="MergeHistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/icons/resources/logo_320x320.png</normaloff>:/icons/resources/logo_320x320.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>From</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="fromNotesList"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="toNotesList"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="diffView"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MergeHistoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MergeHistoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromNotesList</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MergeHistoryDialog</receiver>
   <slot>click_update_from()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toNotesList</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MergeHistoryDialog</receiver>
   <slot>click_update_to()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click_update_from()</slot>
  <slot>click_update_to()</slot>
 </slots>
</ui>
